### PR TITLE
[FIX] website: fix various issues within s_text_cover snippet

### PR DESCRIPTION
**Pull request by chgo-odoo**
- requires https://github.com/odoo/design-themes/pull/873
task-4119904
part of task-3619705

-------

This commit addresses different issues within the `s_text_cover` snippet:

- Latency when previewing the snippet within design-themes due to
a missing `o_colored_level` class on the section ;

- Broken layout in columns mode due to a wronge `col-lg-*` class ;

- Weird visual result in columns mode due to the column padding on the
image (addressed by switching the image back to a background-image) ;

- Unneeded `background-position` CSS, resulting in a weird alignment
of the image withing some themes.

- Image file extremely larger than the snippet size.

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by bvr-odoo — CHANGES_REQUESTED** [review]
Hello @chgo-odoo A few changes to make :wink:

File: addons/website/views/snippets/s_text_cover.xml [hunk 1]
```diff
@@ -2,17 +2,15 @@
 <odoo>
 
 <template id="s_text_cover" name="Text Cover">
-    <section class="s_text_cover o_cc o_cc2">
+    <section class="s_text_cover o_colored_level o_cc o_cc5">
```
[the remark this answers is not in the document]
Comment (chgo-odoo) [reply addons/website/views/snippets/s_text_cover.xml:5]: IIRC we need to keep at least the one on the section. I don't know if you experienced that already but there's an issue if we "nest" multiple `o_cc` without using `o_colored` level at least on the section. When rendered into the snippet modal, the snippet will have some "invisible" elements or at least slow to load. For some reason, using `o_colored_level` works.

Note that we might be able to remove it from the two other occurrences you mentioned, indeed, but I'd keep this one to ensure it is rendered correctly within the modal.

[the remark this answers is not in the document]
Comment (bvr-odoo) [reply addons/website/views/snippets/s_text_cover.xml:5]: @qsm-odoo What do you think ?
If `o_colored_level` is added to sections when section are dropped, should we adapt the JS code to add it to sections  in the snippet modal, or adding it to the XML is enough? Actually, why the class is added in JS and not already in the XML like the `o_cc` classes ?

[the remark this answers is not in the document]
Comment (bvr-odoo) [reply addons/website/views/snippets/s_text_cover.xml:5]: ok thanks for the explanation :+1:  
@chgo-odoo ping me when the PR is ready :+1: I think there is still a small change to make
Thanks :wink:

**Comment by chgo-odoo** [comment]
Yes @bvr-odoo, switched `o_cc5` to `o_cc3` and adapted the DT counterpart as well, it should be ready

**Comment by bvr-odoo** [comment]
Commit(s) moved into

- https://github.com/odoo/odoo/pull/180736

**Pull request closed without merging**